<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>720</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(96, 125, 139);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>1241</width>
      <height>141</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(55, 71, 79)</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>30</y>
       <width>401</width>
       <height>71</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 87 60pt &quot;Arial Black&quot;;
color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string>IN.VOICE</string>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="frame_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>160</y>
      <width>1241</width>
      <height>521</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(207, 216, 220)</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>20</y>
       <width>221</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 87 24pt &quot;Arial Black&quot;;
color: rgb(55, 71, 79)</string>
     </property>
     <property name="text">
      <string>Target Voice</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>230</y>
       <width>221</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 87 24pt &quot;Arial Black&quot;;
color: rgb(55, 71, 79)</string>
     </property>
     <property name="text">
      <string>User Voice</string>
     </property>
    </widget>
    <widget class="QFrame" name="frame_5">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>70</y>
       <width>871</width>
       <height>151</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(55, 71, 79)</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QFrame" name="frame_7">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>851</width>
        <height>131</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QWidget" name="verticalLayoutWidget">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>851</width>
         <height>131</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="layout_target_voice"/>
      </widget>
     </widget>
    </widget>
    <widget class="QFrame" name="frame_6">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>280</y>
       <width>871</width>
       <height>151</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(55, 71, 79)</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QFrame" name="frame_8">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>851</width>
        <height>131</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QWidget" name="verticalLayoutWidget_2">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>851</width>
         <height>131</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="layout_user_voice"/>
      </widget>
     </widget>
    </widget>
    <widget class="QFrame" name="frame_3">
     <property name="geometry">
      <rect>
       <x>930</x>
       <y>0</y>
       <width>311</width>
       <height>521</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(38, 50, 56);</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QFrame" name="frame_4">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>291</width>
        <height>501</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(69, 90, 100)</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QPushButton" name="push_btn_convert">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>271</width>
         <height>91</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(120, 144, 156);
font: 87 20pt &quot;Arial Black&quot;;
color: rgb(255, 255, 255);</string>
       </property>
       <property name="text">
        <string>CONVERT</string>
       </property>
      </widget>
      <widget class="QPushButton" name="push_btn_process">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>110</y>
         <width>271</width>
         <height>91</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(120, 144, 156);
font: 87 20pt &quot;Arial Black&quot;;
color: rgb(255, 255, 255);</string>
       </property>
       <property name="text">
        <string>PROCESS</string>
       </property>
      </widget>
      <widget class="QPushButton" name="push_btn_batch">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>210</y>
         <width>271</width>
         <height>91</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(120, 144, 156);
font: 87 20pt &quot;Arial Black&quot;;
color: rgb(255, 255, 255);</string>
       </property>
       <property name="text">
        <string>BATCH</string>
       </property>
      </widget>
      <widget class="QPushButton" name="push_btn_load">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>310</y>
         <width>271</width>
         <height>91</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(120, 144, 156);
font: 87 20pt &quot;Arial Black&quot;;
color: rgb(255, 255, 255);</string>
       </property>
       <property name="text">
        <string>LOAD</string>
       </property>
      </widget>
      <widget class="QPushButton" name="push_btn_record">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>410</y>
         <width>131</width>
         <height>81</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(120, 144, 156);
font: 87 20pt &quot;Arial Black&quot;;
color: rgb(255, 255, 255);</string>
       </property>
       <property name="text">
        <string>REC</string>
       </property>
      </widget>
      <widget class="QPushButton" name="push_btn_stop">
       <property name="geometry">
        <rect>
         <x>150</x>
         <y>410</y>
         <width>131</width>
         <height>81</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(120, 144, 156);
font: 87 20pt &quot;Arial Black&quot;;
color: rgb(255, 255, 255);</string>
       </property>
       <property name="text">
        <string>STOP</string>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QPushButton" name="push_btn_tgt_sel">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>450</y>
       <width>75</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 87 18pt &quot;Arial Black&quot;;
background-color: rgb(55, 71, 79);
color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string>TGT</string>
     </property>
    </widget>
    <widget class="QPushButton" name="push_btn_usr_sel">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>450</y>
       <width>75</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 87 18pt &quot;Arial Black&quot;;
background-color: rgb(55, 71, 79);
color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string>USR</string>
     </property>
    </widget>
    <widget class="QFrame" name="frame_9">
     <property name="geometry">
      <rect>
       <x>500</x>
       <y>450</y>
       <width>251</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(55, 71, 79);</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>10</y>
        <width>231</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 87 18pt &quot;Arial Black&quot;;
color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>SELECT TRACK</string>
      </property>
     </widget>
    </widget>
    <widget class="QFrame" name="frame_10">
     <property name="geometry">
      <rect>
       <x>29</x>
       <y>450</y>
       <width>211</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(120, 144, 156);</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>10</y>
        <width>171</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 87 18pt &quot;Arial Black&quot;;
color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>SELECT SRC</string>
      </property>
     </widget>
    </widget>
    <widget class="QFrame" name="frame_11">
     <property name="geometry">
      <rect>
       <x>750</x>
       <y>450</y>
       <width>151</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(69, 90, 100)</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QLineEdit" name="line_edit_id_sel">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>131</width>
        <height>31</height>
       </rect>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 87 18pt &quot;Arial Black&quot;;
color: rgb(255, 255, 255);
background-color: rgb(120, 144, 156);
border: none;</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="placeholderText">
       <string>HIT ENT</string>
      </property>
     </widget>
    </widget>
    <widget class="QPushButton" name="push_btn_tgt_play">
     <property name="geometry">
      <rect>
       <x>810</x>
       <y>20</y>
       <width>91</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 87 18pt &quot;Arial Black&quot;;
color: rgb(255, 255, 255);
background-color: rgb(55, 71, 79)</string>
     </property>
     <property name="text">
      <string>PLAY</string>
     </property>
    </widget>
    <widget class="QPushButton" name="push_btn_usr_play">
     <property name="geometry">
      <rect>
       <x>810</x>
       <y>230</y>
       <width>91</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 87 18pt &quot;Arial Black&quot;;
color: rgb(255, 255, 255);
background-color: rgb(55, 71, 79)</string>
     </property>
     <property name="text">
      <string>PLAY</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1280</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
